apps/graphql/fixtures/addressSearch.js

```diff
@@ -0,0 +1,87 @@
+const NO_ADDITIONS = "NO_ADDITIONS"; // address without additions, exact match
+const VALID_WITH_ADDITION = "VALID_WITH_ADDITION"; // number is valid in combination with addition
+const VALID_WITHOUT_ADDITION = "VALID_WITHOUT_ADDITION"; // number is valid without an addition
+const MANY_ADDITIONS = "MANY_ADDITIONS"; // many additions available
+const EXACT_MATCH = "EXACT_MATCH"; // Only one address for given housenumber + zipcode, ie 26 b and no 26
+
+const HAS_NOUN_ADDITION = "HAS_NOUN_ADDITION"; // One or more a, b, etc. (a char)
+const HAS_NUM_ADDITION = "HAS_NUM_ADDITION"; // One or more 1 1, 1 2, etc. (an int)
+const HAS_ALPHANUM_ADDITION = "HAS_ALPHANUM_ADDITION"; // One or more a 1, b 2, rood etc. (a string)
+const HAS_PLURAL_NOUN_ADDITION = "HAS_PLURAL_NOUN_ADDITION"; // One or more 20 H L, 20 H R
+
+const IS_METRO_STATION = "IS_METRO_STATION";
```

Comment:
I'm not really sure what the purpose is with the Metro station... maybe we can remove this for now.